riscv/processor.cc

```diff
@@ -970,6 +992,13 @@ const char* processor_t::get_symbol(uint64_t addr)
   return sim->get_symbol(addr);
 }
 
+void processor_t::execute_insn_prehook(insn_t insn)
+{
+  if (extension_enabled(EXT_ZICFILP)) {
+    zicfilp_maybe_expect_lpad(state.elp, insn);
+  }
```

Comment:
`zicfilp_maybe_expect_lpad` could be renamed as `zicfilp_check_if_lpad_required` to make it slightly more clear what its doing.

Reply:
Fixed